connector/connect/server/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -703,21 +703,13 @@ package object dsl {
       def drop(columns: String*): Relation = {
         assert(columns.nonEmpty)
 
-        val cols = columns.map(col =>
-          Expression.newBuilder
-            .setUnresolvedAttribute(
-              Expression.UnresolvedAttribute.newBuilder
-                .setUnparsedIdentifier(col)
-                .build())
-            .build())
-
         Relation
           .newBuilder()
           .setDrop(
             Drop
               .newBuilder()
               .setInput(logicalPlan)
-              .addAllCols(cols.asJava)
+              .addAllColumnNames(columns.toSeq.asJava)
```

Comment:
maybe also add a `def drop(cols: Column*)`?

Reply:
do you mean introducing a new public api? I don't know, maybe need more discussion

Reply:
NVM this is at least as good as before.